NotificationService/NotificationService.Contracts/Entities/EmailNotificationMapEntity.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace NotificationService.Contracts.Entities
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using Microsoft.Azure.Cosmos.Table;
+
+    /// <summary>
+    /// Storage Entity to store EmailId and NotificationId mapping for GDPR implementation.
+    /// </summary>
+    public class EmailNotificationMapEntity : TableEntity
+    {
+        /// <summary>
+        /// Gets or Sets ApplicationName.
+        /// </summary>
+        public string ApplicationName { get; set; }
```

Comment:
Where is email ID, NotificationID, To/cc/bcc?

Reply:
to cc and bcc is converted to emailId and ntoficationid mapping like 
if for notificationId 12345, from = abc@test.com, cc= efg@test.com;hij@test.com, bcc= klm@test.com, to= nop@test.com
then there will be a row for each email id in table like
partitionkey      rowkey 
abc@test.com   12345
efg@test.com   12345
hij@test.com     12345
klm@test.com   12345
nop@test.com   12345
so no additional cc/bcc etc are required. when scrubbing  an email we will get corresponding notificationIds and then do scrub.